ruby/ql/lib/ruby.qll

```diff
@@ -3,5 +3,5 @@
  */
 
 import codeql.ruby.AST as Ast
-import codeql.ruby.CFG as Cfg
+import codeql.ruby.CFG
```

Comment:
I'm not a fan of bringing everything from `codeql.ruby.CFG` into the default namespace; Do you remember where things went wrong? I tried reverting this line and the changes to `WeakSensitiveDataHashingCustomizations.qll`, and with that the query compiles just fine.

Reply:
Alright, I think I found most cases of this, so here's an alternative commit I could swap in: https://github.com/github/codeql/pull/20318/commits/b548685b16bae166fcb98a1d78cab0c169849924
So you prefer that or something else?

Reply:
Done.